packages/block-library/src/comments-query-loop/toolbar.js

```diff
@@ -49,6 +55,19 @@ export default function CommentsQueryLoopToolbar( { perPage, setQuery } ) {
 								isDragEnabled={ false }
 							/>
 						</BaseControl>
+						<ToggleControl
+							label={ __( 'Newer comments first' ) }
+							checked={ order === 'desc' || order === null } // Settings value not available on REST API.
+							onChange={ () => {
+								setAttributes( {
+									...attributes,
+									order:
+										order === 'desc' || order === null
+											? 'asc'
+											: 'desc',
```

Comment:
Would it work if we flip the check?
```suggestion
									order: order !== 'asc' ? 'desc" : 'asc',
```